### PR TITLE
Improve/add init script on start lhasa

### PR DESCRIPTION
**Pull request by yroffin**
When lhasa start (with appcatalog.sh)
If exist on file call init-script.sh then execute it (like mycompany.sh) to permit custom configuration on lhasa integrators

### CODE REVIEW

File: scripts/appcatalog.sh [hunk 1]
```diff
@@ -4,18 +4,31 @@ set -ex
 
 ./appcatalog $* &
 
-[ "x${IMPORT_SAMPLE_DATA}" != "x" ] && {
+wait_for_apis () {
     export API_BASE_URL=http://localhost:8081/api
     # wait for api
     ret=1
     while [ $ret -ne 0 ]
```
Comment (rayene) [inline scripts/appcatalog.sh]: Can we limit the waiting to 60 seconds ?

File: scripts/appcatalog.sh [hunk 2]
```diff
@@ -4,18 +4,31 @@ set -ex
...
+[ "x${IMPORT_SAMPLE_DATA}" != "x" ] && {
+    wait_for_apis
     ./mycompany.sh 2>/dev/null >/dev/null
 }
 
+# Any script called init-script.sh will be executed
+# Use it for your own integration
+[ -f ./init-script.sh ] && {
+    wait_for_apis
+    chmod 700 ./init-script.sh && ./init-script.sh 2>/dev/null >/dev/null
```
Comment (rayene) [inline scripts/appcatalog.sh]: /dev/null hides a lot of useful stuff. No ?

File: scripts/appcatalog.sh (hunk 1 above)
Comment (yroffin) [reply scripts/appcatalog.sh]: Yes but we must abort the container no ?

Comment (yroffin) [reply scripts/appcatalog.sh]: My be in a next PR ... but for me we must wait infinitely

File: scripts/appcatalog.sh (hunk 2 above)
Comment (yroffin) [reply scripts/appcatalog.sh]: done

**Comment by yroffin** [comment]
On voit demain j’ai trouvé mon bug de déploiement ;)

Salut

File: scripts/appcatalog.sh [hunk 3]
```diff
@@ -1,21 +1,34 @@
...
+
+# Any script called init-script.sh will be executed
+# Use it for your own integration
+[ -f ./init-script.sh ] && {
+    wait_for_apis
+    chmod 700 ./init-script.sh && ./init-script.sh
 }
 
 wait %1
+pwd
```
Comment (FabienM) [inline scripts/appcatalog.sh]: Remove this

File: scripts/appcatalog.sh [hunk 4]
```diff
@@ -1,21 +1,34 @@
...
 
-[ "x${IMPORT_SAMPLE_DATA}" != "x" ] && {
+wait_for_apis () {
     export API_BASE_URL=http://localhost:8081/api
     # wait for api
     ret=1
     while [ $ret -ne 0 ]
     do
-        curl -s $API_BASE_URL/unsecured/version >/dev/null
+        curl $API_BASE_URL/unsecured/version
```
Comment (FabienM) [inline scripts/appcatalog.sh]: Should be muted?

Comment (yroffin) [reply scripts/appcatalog.sh]: Done

File: scripts/appcatalog.sh (hunk 3 above)
Comment (yroffin) [reply scripts/appcatalog.sh]: Done

**Pull request merged**